demo-1-before.sh

```
@@ -0,0 +1,220 @@
+#!/usr/bin/env bash
+# ─────────────────────────────────────────────────────────────────────────────
+# demo-1-before.sh
+# Tear down any existing cluster and build the BEFORE state:
+#   k3d cluster (4 nodes) + Istio + Prometheus + Loki + Tempo
+#   + Robot Shop (with Istio sidecars) + HotROD
+#   NO OTel Collector.
+#
+# Run from: ~/Documents/Claude/Projects/sre-stack
+# ─────────────────────────────────────────────────────────────────────────────
+set -uo pipefail
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)